.nvmrc

```diff
@@ -0,0 +1 @@
+v18.19.0
```

Comment:
I've checked what upstream does in other projects, they seldom add the nvmrc file, so I am sure what to do about it: https://github.com/search?q=org%3Adhis2%20nvmrc&type=code

I see in any case the full version "v20", so let's do that, put that version and check that everything works (lint, test, build)

Reply:
Updated. Lint, test and build works with v20.